CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
     - `-alertmanager.alertmanager-client.grpc-max-recv-msg-size` now defaults to 100 MiB (previously was not configurable and set to 16 MiB)
     - `-alertmanager.alertmanager-client.grpc-max-send-msg-size` now defaults to 100 MiB (previously was not configurable and set to 4 MiB)
     - `-alertmanager.max-recv-msg-size` now defaults to 100 MiB (previously was 16 MiB)
+* [CHANGE] Dashboards: Remove per-user series legends from Tenants dashboard. #1605
```

Comment:
Wait, this should be below in the `### Mixin` section instead.

Reply:
Ah, good catch, thanks